sympy/sets/sets.py

```diff
@@ -2531,3 +2635,52 @@ def set_pow(x, y):
 def set_function(f, x):
     from sympy.sets.handlers.functions import _set_function
     return _set_function(f, x)
+
+class SetKind(Kind):
```

Comment:
There should usually be two blank lines between classes or function etc that are defined at top-level.